engine/src/main/resources/org/operaton/bpm/engine/db/upgrade/h2_engine_7.22_to_7.23.sql

```diff
@@ -0,0 +1,19 @@
+--
+-- Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
```

Comment:
How do we deal with new files? When they are copied from existing it might be necessary to keep the original copyright notice, but somehow you should be incorporated.

Same for all other new files.

Reply:
this PR is actually a backported commit, so in this case we would have to leave the copyright notice as it is